src/abstracts/Command.ts

```diff
@@ -12,7 +12,7 @@ abstract class Command {
 		this.options = options;
 	}
 
-	abstract async run(message: Message, args?: string[]): Promise<void>;
+	abstract run(message: Message, args?: string[]): Promise<void>;
```

Comment:
What was the reasoning behind removing `async` here?

Reply:
I did this, typescript doesn't allow `async` to be used with `abstract`

> TS1243: 'async' modifier cannot be used with 'abstract' modifier.